CodeStarterCamp_Week3/step1.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  step1.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by MARY on 2023/03/07.
+//
+
+import Foundation
+
+struct Person {
+    let name: String
+    var money: Int
+    func order(_ coffee: Coffee, at coffeshop: CoffeeShop) {
```

Comment:
일반적으로 프로퍼티와 메서드 사이는 개행을 한번 해줍니다 ㅎㅎ
```suggestion
    var money: Int
    
    func order(_ coffee: Coffee, at coffeshop: CoffeeShop) {
```